products/workflows/manifest.tsx

```diff
@@ -63,7 +63,6 @@ export const manifest: ProductManifest = {
             path: 'Workflows',
             href: urls.workflows(),
             type: 'workflows',
-            visualOrder: PRODUCT_VISUAL_ORDER.workflows,
```

Comment:
Hmm?

Reply:
Not being used anywhere, removed to simplify my mental model - we're always sorting alphabetically ha (both categories and products)

We can add this back if we ever wanted to add ordering (which I don't think we need since we'll be hiding all products anyway, ordering won't matter)